calibration/FLORDM/CGINS-FLORDM-03124__20190827.csv

```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+FLBBSLC-3124,CC_angular_resolution,1.096,
+FLBBSLC-3124,CC_depolarization_ratio,0.039,
+FLBBSLC-3124,CC_measurement_wavelength,700.0,
+FLBBSLC-3124,CC_scattering_angle,140.0,
```

Comment:
FYI to reviewers: You may notice the SN for this sensor (FLBBSLC-3124) is different than all other sensors. It was originally entered in bulkload with the FLBBSLC appended and that's how it will have to be referenced in CAL csvs until we update bulk load.